RecoBTag/Combined/python/caloDeepCSVJetTags_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+caloDeepCSVJetTags = cms.EDProducer(
+	'DeepFlavourJetTagsProducer',
+	src = cms.InputTag('caloDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
+	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json'),
+  toAdd = cms.PSet(
+      probcc = cms.string('probc')
+      ),
+	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(caloDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
```

Comment:
Even though calo BTag sequence will be used solely at HLT but we use calo BTag sequence as a prefilter followed by PF Reco + PF BTag sequence. So we prefer to keep both (calo & pf) sequences in sync to avoid any confusion to PAG HLT path developers. 

Moreover, we will be using "offline PF DeepCSV training" for "calo as well as pf Jets at HLT" so we prefer to keep the era based structure as dedicated trainings in HLT land did not yield any different results compared to use of offline training.

We are also working on dedicated Phase-2 retraining so we wish to stick with era structure in place.